src/Og.php

```diff
@@ -128,6 +128,34 @@ public static function createField($plugin_id, $entity_type, $bundle, array $set
   }
 
   /**
+   * Get the parent groups of an entity.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity objects.
+   *
+   * @return array
+   *   Array of group entities.
+   */
+  public static function getParentGroups(EntityInterface $entity) {
```

Comment:
Maybe we could move this to an `OgSubgroups.php`, as it will not be as used as `Og.php`